File: app/src/main/java/com/qwert2603/vkmessagestat/di/AppModule.java
package com.qwert2603.vkmessagestat.di;

import android.content.Context;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Module
public class AppModule {

    private Context mAppContext;

    public AppModule(Context appContext) {
        mAppContext = appContext;
    }

    @Provides
    @Singleton
    Context mContext() {
        return mAppContext;
    }

}

File: app/src/main/java/com/qwert2603/vkmessagestat/di/AppModule.kt
package com.qwert2603.vkmessagestat.di

import android.content.Context
import dagger.Module
import dagger.Provides
import javax.inject.Singleton

@Module
class AppModule(private val mAppContext: Context) {

    @Provides
    @Singleton
    internal fun mContext(): Context {
        return mAppContext
    }

}
